app/main.py

```diff
@@ -2,0 +1,43 @@
+class Animal:
+    def __init__(self, name, appetite, is_hungry=True):
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+    
+    def print_name(self):
+        print(f"Hello, I'm {self.name}.")
+    
+    def feed(self):
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points . . .")
+            self.is_hungry = False
+            return self.appetite
+        else:
+            return 0
+
+class Cat(Animal):
+    def __init__(self, name, is_hungry=True):
+        self.name = name
+        self.is_hungry = is_hungry
+        super().__init__(appetite=3)
+    
+    @staticmethod
+    def cath_mouse():
```

Comment:
The method name `cath_mouse` is misspelled. It should be `catch_mouse` to match the task description.